index.html

```diff
@@ -15,10 +15,25 @@
   </head>
   <body>
     <div class="container">
+      <div class="show-hamburger hidden">
+        <div class="cross-container">
+          <img
+            src="./assets/icons/cancelHamburger.png"
+            alt=""
+            class="close-cross"
+          />
+        </div>
+        <ul class="show-links-hamburger">
+          <li><a href="#portfolio" class="menu-links">Portfolio</a></li>
+          <li><a href="#about-info" class="menu-links">About</a></li>
+          <li><a href="#contact" class="menu-links">Contact</a></li>
+        </ul>
```

Comment:
- Excellent job! :clap: :clap: All the functions are well implemented. :vulcan_salute: 
- [ ] However, we do have a few disparity in your choice of styling when compared with the Figma design. There seem to be too much `padding` and `margin-top` on your navigation links. I found a `padding: 32px` and a `margin-top: 24px` on each while all you have on the Figma is just a spacing value of `24px`/`1.5rem`. See the Image below:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/223208880-a727151b-db2b-43cf-98af-4f02b07eb2ca.png)|![image](https://user-images.githubusercontent.com/99715640/223208967-51a3263b-0283-4c6d-9611-368297e0b67d.png)|
- Kindly fix this to match design, and to also improve your user interface, as well as the experience users have with accessing your page :wink: